docs/native.rst

```diff
@@ -76,3 +76,64 @@ To implement a user-defined model, implement your model as a Python function, an
 :meth:`~manticore.native.State.invoke_model` documentation for more. The
 `core models <https://github.com/trailofbits/manticore/blob/master/manticore/models.py>`_
 are also good examples to look at and use the same external user API.
+
+Symbolic Input
+--------------
+
+Manticore allows you to execute programs with symbolic input, which represents a range of possible inputs. You
+can do this in a variety of manners.
+
+Wildcard byte
+^^^^^^^^^^^^^
+
+Throughout these various interfaces, the '+' character is defined to designate a byte
+of input as symbolic. This allows the user to make input that mixes symbolic and concrete
+bytes (e.g. known file magic bytes).
+
+For example: ``"concretedata++++++++moreconcretedata++++++++++"``
+
+Symbolic arguments/environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To provide a symbolic argument or environment variable on the command line,
+use the wildcard byte where arguments and environment are specified.::
+
+    $ manticore ./binary +++++ +++++
+    $ manticore ./binary --env VAR1=+++++ --env VAR2=++++++
+
+For API use, use the ``argv`` and ``envp`` arguments to the :meth:`manticore.native.Manticore.linux` classmethod.::
+
+    Manticore.linux('./binary', ['++++++', '++++++'], dict(VAR1='+++++', VAR2='++++++'))
+
+Symbolic stdin
+^^^^^^^^^^^^^^
+
+Manticore by default is configured with 256 bytes of symbolic stdin data which is configurable
+with the ``stdin_size`` kwarg of :meth:`manticore.native.Manticore.linux` , after an optional
+concrete data prefix, which can be provided with the ``concrete_start`` kwarg of
+:meth:`manticore.native.Manticore.linux`.
```

Comment:
hmm, seems like some mismatch with ReST format?
![image](https://user-images.githubusercontent.com/10009354/60181925-4ad69480-97f1-11e9-800d-b185284f5d37.png)

Reply:
Where are you seeing this?

https://manticore.readthedocs.io/en/latest/native.html#symbolic-input

![image](https://user-images.githubusercontent.com/12704613/60191072-b88abc80-9801-11e9-827f-5b4daf038cbc.png)

Reply:
Oh it renders there properly. I checked it via github, my bad then. LGTM :)